### DISCUSSION TITLE
display inside of HA

### DISCUSSION THREAD

**Discussion by theculby5** [music-assistant/support#1452] [Q&A] [answered]
I have been getting MA setup inside of my HA. I am trying to move away from the Alexa devices and are using the AudioCast M5 devices connected to speakers. My question is that I am using FireTablets to display HA and the Music Assistant interface doesn't show up. Is there something that I am doing wrong?

**Comment by krisnoble** [comment, marked as answer]
It sounds like you're expecting the old behaviour, you need to open the MA interface separately on port 8095 now. You could add it via an [iframe panel](https://www.home-assistant.io/integrations/panel_iframe/)?
